CMakeLists.txt

```
@@ -33,6 +37,19 @@ set(CPACK_NSIS_DISPLAY_NAME ${APPLICATION_NAME})
 set(CPACK_NSIS_INSTALLED_ICON_NAME ${EXECUTABLE_NAME}.exe)
 set(CPACK_NSIS_EXECUTABLES_DIRECTORY .)
 set(CPACK_NSIS_MUI_FINISHPAGE_RUN ${EXECUTABLE_NAME}.exe)
+
+# Setup installer exe properties
+set (CPACK_NSIS_EXTRA_DEFINES 
+   VIProductVersion \"${VERSION}\"
+   VIFileVersion    \"${VERSION}.0\"
+   VIAddVersionKey  \"ProductName\" \"${APPLICATION_NAME}\"
+   VIAddVersionKey  \"ProductVersion\" \"${VERSION}\"
+   VIAddVersionKey  \"CompanyName\" \"${COMPANY_NAME}\"
+   VIAddVersionKey  \"LegalCopyright\" \"${LEGAL_COPYRIGHT}\"
+   VIAddVersionKey  \"FileDescription\" \"${APPLICATION_NAME} Installer\"
+   VIAddVersionKey  \"FileVersion\" \"${VERSION}\"
+   VIAddVersionKey \"InternalName\" \"${APPLICATION_NAME}\"
+)
```

Could you adjust formatting?
```suggestion
set(CPACK_NSIS_EXTRA_DEFINES 
   VIProductVersion \"${VERSION}\"
   VIFileVersion \"${VERSION}.0\"
   VIAddVersionKey \"ProductName\" \"${APPLICATION_NAME}\"
   VIAddVersionKey \"ProductVersion\" \"${VERSION}\"
   VIAddVersionKey \"CompanyName\" \"${COMPANY_NAME}\"
   VIAddVersionKey \"LegalCopyright\" \"${LEGAL_COPYRIGHT}\"
   VIAddVersionKey \"FileDescription\" \"${APPLICATION_NAME} Installer\"
   VIAddVersionKey \"FileVersion\" \"${VERSION}\"
   VIAddVersionKey \"InternalName\" \"${APPLICATION_NAME}\"
)
```

---

I aligned it for a better readibility

---

I see, but I would refer to avoid alignment.

---

As you like. 
For me alignment is better to check easiliy text strings.